engines/wintermute/base/gfx/opengl/mesh3ds.cpp

```diff
@@ -85,15 +63,14 @@ bool Wintermute::Mesh3DS::loadFrom3DS(Common::MemoryReadStream &fileStream) {
 		switch (chunkId) {
 		case VERTICES:
 			_vertexCount = fileStream.readUint16LE();
-			_vertexData = new byte[4 * 3 * _vertexCount];
+			_vertexData = new GeometryVertex[_vertexCount];
```

Comment:
Add trailing () to zero-initialize the allocation.

Reply:
Done